starctl

```diff
@@ -132,7 +132,7 @@ if [[ ${#STARGATE_ARGS[@]} -eq 0 ]]; then
         STARGATE_ARGS+=("--proxy-port" "$PROXY_PORT")
     fi
 
-    if [[ ! -z "$DEVELOPER_MODE" ]]; then
+    if [[ ! -z "$DEVELOPER_MODE" ]] && [ "$DEVELOPER_MODE" = true ]; then
```

Comment:
`DEVELOPER_MODE=1` should probably work too.

Reply:
Agree

Reply:
I think `[[ ! -z "$DEVELOPER_MODE" ]]` is not needed now that we have  `&&` and a test for a non-empty value. 

Also the second check should probably use double brackets for consistency with other `if` statements.

Reply:
Double bracket and single bracket tests in bash always trip me up.